srcpkgs/linux6.10/files/i386-dotconfig

```diff
@@ -2916,7 +2916,7 @@ CONFIG_TUN_VNET_CROSS_LE=y
 CONFIG_VETH=m
 CONFIG_VIRTIO_NET=m
 CONFIG_NLMON=m
-# CONFIG_NETKIT is not set
+CONFIG_NETKIT=y
```

Comment:
I'm not sure I'd bother with this one. Added it for consistency's sake?

Reply:
Pretty much.